package.json

```diff
@@ -44,6 +44,7 @@
     "create:cmp": "plop component",
     "create:pkg": "plop package",
     "create:hook": "plop hook",
+    "changeset:beta": "changeset pre enter beta",
```

Comment:
@wingkwong should not we have a `changeset pre enter canary` as well?

Reply:
@jrgarciadev `canary` is not a pre-release but a snapshot release. By definition, snapshot releases are a way to release your changes for testing without updating the versions. In canary, we don't care about the versions but the snapshot of the changes in canary branch at a specific moment.